Configs/.local/lib/hyde/cyclewallpaperandtheme.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env sh
+
+# Paths to scripts and directories
+WALL_SCRIPT="$HOME/.local/share/bin/swwwallpaper.sh"
+THEME_SCRIPT="$HOME/.local/share/bin/themeswitch.sh"
+#SDDM_SCRIPT="$HOME/bin/setsddmwallpaper"
+SDDM_SCRIPT="$HOME/.local/share/bin/setsddmwallpaper.sh"
+LOG_FILE="$HOME/.local/share/cyclewallpaperandtheme.log"
+WALL_CYCLE_FILE="$HOME/.local/share/cyclewallpapercount"
+THEME_CYCLE_FILE="$HOME/.local/share/cyclethemecount"
+THEME_CHANGE_FLAG_FILE="$HOME/.local/share/cyclethemeflag"
+THEMES_DIR="$HOME/.config/hyde/themes"
+HYDE_CONFIG="$HOME/.config/hyde/hyde.conf"
```

Comment:
Too hardcoded, I will expose a global variables later and will use it so global stuff can be source/imported. Also don't put logs in .local/share.